src/UmbracoExamine.PDF/PdfTextService.cs

```diff
@@ -13,17 +14,32 @@ public class PdfTextService
     {
         private readonly IPdfTextExtractor _pdfTextExtractor;
         private readonly IMediaFileSystem _mediaFileSystem;
+        private readonly ILogger _logger;
 
-        public PdfTextService(IPdfTextExtractor pdfTextExtractor, IMediaFileSystem mediaFileSystem)
+        public PdfTextService(
```

Comment:
Just for backwards compat, can you obsolete this method and have a newer overload? Then you can pass `Current.Logger` into the new overload

Reply:
Oh yes, great idea. This is done.  At the time I started the text processor there wasn't a good alternative to doing it myself, now there is. I hope I can carve out some time to help with some other things.